fixtures/colorful-component.fixtures.ts

```diff
@@ -1,37 +1,36 @@
 /* eslint-disable import/no-anonymous-default-export */
 
-export type DifferentBgColor = "blueColor" | "greenColor" | "yellowColor";
+export type DifferentBgColor = "blue-color" | "green-color" | "yellow-color";
 
-export type bgColorTtitle = "blueColor" | "greenColor" | "yellowColor";
+export type bgColorTtitle = "blue-color" | "green-color" | "yellow-color";
 
-export interface IColorfullComponent {
+export interface IColorfulComponent {
   textTitle: string
   bgColor?: DifferentBgColor
   bgTitle?: bgColorTtitle
```

Comment:
Place this inline too. Next time consider that types and interfaces shouldn't be `camelCased`